package.json

```diff
@@ -11,11 +11,14 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@types/styled-components": "^5.1.0",
```

Comment:
maybe move this to devdependencies, since we don't directly import types for styled-components.